### PR TITLE
feature/AT-SCE-UI_AdminOperations_deleteUser

### PR DESCRIPTION
**Pull request by FlavioAlejandro**
Added changes to "admin.controller.js" and "at-sce-api.service.js" to integrate the delete method "deleteUser" and "UpdateUser".

### CODE REVIEW

File: src/controllers/admin.controller.js [hunk 1]
```diff
@@ -145,19 +152,92 @@ adminCtrl.updateUser = async (req, res) => {
...
+      type: parseInt(user_type),
+      firstName: user_firstName,
+      lastName: user_lastName,
+      email: user_email,
+      status: parseInt(user_status),
+    };
+    console.log(request);
+    // Call Update USER - PUT /api/v1/users endpoint
+    await sceServiceAPI.updateUser(request).then((result) => {
+      console.log("Resultado-->", result);
```
Comment (at-internship) [inline src/controllers/admin.controller.js]: Comment this line

File: src/controllers/admin.controller.js [hunk 2]
```diff
@@ -145,19 +152,92 @@ adminCtrl.updateUser = async (req, res) => {
...
+    // Request
+    let request = {
+      id: user_id,
+      type: parseInt(user_type),
+      firstName: user_firstName,
+      lastName: user_lastName,
+      email: user_email,
+      status: parseInt(user_status),
+    };
+    console.log(request);
```
Comment (at-internship) [inline src/controllers/admin.controller.js]: Comment this line

File: src/views/admin/user/add-users.hbs [hunk 3]
```diff
@@ -0,0 +1,214 @@
+<div class="d-flex flex-column flex-md-row align-items-center p-2 px-md-4 mb-3 bg-white border-bottom box-shadow">
```
Comment (at-internship) [inline src/views/admin/user/add-users.hbs]: This file add-user.hbs should not be part of this PR

**Review by at-internship — APPROVED** [review]
Please review code  comments

**Review by DianaArceoSuarez — APPROVED** [review]

**Review by MiltonNAR — APPROVED** [review]

**Review by jaimeIxba — APPROVED** [review]

**Review by AT-ivillamil — APPROVED** [review]

**Review by irvingesp2190 — APPROVED** [review]

File: src/views/admin/user/add-users.hbs (hunk 3 above)
Comment (at-internship) [inline src/views/admin/user/add-users.hbs]: Delete this file

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/views/admin/user/add-users.hbs (hunk 3 above)
Comment (FlavioAlejandro) [reply src/views/admin/user/add-users.hbs]: Done

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**